integrationTests/webpack/webpack.config.json

```diff
@@ -2,6 +2,8 @@
   "mode": "production",
   "entry": "./entry.js",
   "output": {
-    "libraryTarget": "commonjs2"
+    "library": {
+      "type": "commonjs2"
+    }
```

Comment:
Is it a fix?
If so please extract it in a separate PR and provide an explanation for it.

Reply:
![image](https://user-images.githubusercontent.com/8672915/140650535-6b852992-b19b-49e8-b66a-a1cbc0bb43e5.png)


I will remove it, but I won't make a separate PR just for

Reply:
It's totally worth it, same as any other unrelated change.
Every change that makes sense on its own should be a separate PR.